Apsitvarkom.UnitTests/DataAccess/PollutedLocationDTOFileRepositoryTests.cs

```diff
@@ -270,5 +270,21 @@ public async Task GetByPropertyAsync_SeveralInstanceWithRequestedIdFound_FirstMa
 
         Assert.That(instance!.Radius, Is.EqualTo(2));
     }
+
+    [Test]
+    public async Task tempTest()
```

Comment:
you don't need `async Task` here as is in the test above, as the method ReverseGeocode is not async. considering using `void` instead